tests/snappi/pfc/files/helper.py

```diff
@@ -40,7 +40,8 @@ def run_pfc_test(api,
                  prio_dscp_map,
                  test_traffic_pause,
                  headroom_test_params=None,
-                 pfc_pause_src_mac=None):
+                 pfc_pause_src_mac=None,
+                 class_enable_vec=True):
```

Comment:
It seems to me that "set_class_enable_vec" is a better name.

Reply:
Ok, will change